misc/formatter/modules/formatter-core/src/main/java/org/ballerinalang/formatter/core/options/IndentFormattingOptions.java

```diff
@@ -63,7 +63,7 @@ public static class IndentFormattingOptionsBuilder {
         private static final String CONTINUATION_INDENT_SIZE = "continuationIndentSize";
         private int indentSize = getDefaultInt(FormatSection.INDENT, INDENT_SIZE);
         private int continuationIndentSize = getDefaultInt(FormatSection.INDENT, CONTINUATION_INDENT_SIZE);
-        private String wsCharacter = " ";
+        private static final String wsCharacter = " ";
```

Comment:
if we define a variable `static final` we need to make variable name as capital.
`ex:- WS_CHARACTER`